newspack-theme/inc/template-tags.php

```diff
@@ -374,12 +374,20 @@ function newspack_post_thumbnail() {
 	function newspack_comment_form( $order ) {
 		if ( true === $order || strtolower( $order ) === strtolower( get_option( 'comment_order', 'asc' ) ) ) {
 
-			comment_form(
-				array(
-					'logged_in_as' => null,
-					'title_reply'  => null,
-				)
+			$comment_attributes = array(
+				'logged_in_as' => null,
+				'title_reply'  => null,
 			);
+
+			$comment_policy = get_theme_mod( 'comment_policy', '' );
+			$display_policy = get_theme_mod( 'display_comment_policy', false );
+
+			// Check if there's a comment policy set in the Customizer.
+			if ( $display_policy && '' !== $comment_policy ) {
+				$comment_attributes['title_reply_before'] = '<div class="comment-policy">' . wp_kses_post( $comment_policy ) . '</div>';
```

Comment:
This will guard against invalid HTML being entered into the Comment Policy field (source: I entered some by accident 😅):
```suggestion
				$comment_attributes['title_reply_before'] = '<div class="comment-policy">' . wp_kses_post( force_balance_tags( $comment_policy ) ) . '</div>';
```

Reply:
Actually, I think this would be better implemented as part of a custom sanitize callback [here](https://github.com/Automattic/newspack-theme/pull/1205/files#diff-83cc0878126972a709ebefafd5bd74d757c79f98b7309fcc0bbe8845678dc656R954), where you could create a function that runs the input through `wp_kses_post( force_balance_tags( $input ) )`. This way `force_balance_tags` doesn't have to be run on every front-end page load. What do you think?

Reply:
Great catch! I think you just saved me from troubleshooting exploded HTML in the future! I went with the custom sanitation function, added in c5829e2.